### PR TITLE
Updating pandas version requirements 

### PR DESCRIPTION
**Pull request by rpauly18**
-Requiring pandas version v1.0.0 or greater

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-pandas
+pandas>=1.0.0
```
Comment (ssolson) [inline requirements.txt:1]: Do we need greater than 1? I thought the issues were with pandas<0.25?

Comment (rpauly18) [reply requirements.txt:1]: The pandas versions jumped from v0.25 to v1.0, so v1.0 is the first version that will work.

**Comment by rpauly18** [comment]
@ssolson can I merge this?

**Comment by ssolson** [comment]
Yes. Apologies for leaving this hanging. I was just going through all this stuff as I can see the finish line on #53.

**Comment by rpauly18** [comment]
> Yes. Apologies for leaving this hanging. I was just going through all this stuff as I can see the finish line on #53.

Great. I plan to do another review of #53 this week as well.

**Pull request merged**